test/e2e/pivoting.go

```diff
@@ -296,24 +305,22 @@ func installIronicBMO(inputGetter func() installIronicBMOInput) {
 	cmd.Dir = path
 	cmd.Env = append(env, os.Environ()...)
 
-	outputPipe, er := cmd.StdoutPipe()
-	Expect(er).To(BeNil(), "Cannot get the stdout from the command")
-	errorPipe, er := cmd.StderrPipe()
-	Expect(er).To(BeNil(), "Cannot get the stderr from the command")
-	err := cmd.Start()
-	Expect(err).To(BeNil(), "Failed to deploy Ironic")
-	data, er := io.ReadAll(outputPipe)
-	Expect(er).To(BeNil(), "Cannot get the stdout from the command")
-	if len(data) > 0 {
-		Logf("Output of the shell: %s\n", string(data))
-	}
-	errorData, er := io.ReadAll(errorPipe)
-	Expect(er).To(BeNil(), "Cannot get the stderr from the command")
-	err = cmd.Wait()
-	if len(errorData) > 0 {
-		Logf("Error of the shell: %v\n", string(errorData))
+	stdoutStderr, er := cmd.CombinedOutput()
+	Logf("%s\n", stdoutStderr)
+	Expect(er).To(BeNil(), "Failed to deploy Ironic")
+
+	name := "-ironic"
+	if !input.deployIronic {
```

Comment:
Makes more sense to check for BMO I think. Note that both could also be set to true (at least that is how the deploy.sh script works).
```suggestion
	if input.deployBMO {
```

Reply:
done thanks now it should wait for both if the function deployed both at the same time